dataland_analytics/analysis_scripts/scripts_with_api_admin/opened_and_closed_requests.py

```diff
@@ -0,0 +1,42 @@
+import dataland_requests
+from dataland_analytics.config import *
+from dataland_requests.models.request_status import RequestStatus
+
+OBSERVATION_PERIOD_START_EPOCH = 1714588322000
+
+configuration = dataland_requests.Configuration(
+    host="https://dataland.com/community", access_token=PERSONAL_USER_API_KEY
+)
+
+with dataland_requests.ApiClient(configuration) as api_client:
+
+    # Get all requests
+    requests_api_instance = dataland_requests.RequestControllerApi(api_client)
+    requests = requests_api_instance.get_data_requests()
+
+    # Categorize and count requests
+    open_requests = 0
+    closed_requests = 0
+    newly_closed_requests = 0
+    newly_opened_requests = 0
+    for request in requests:
+        if request.creation_timestamp > OBSERVATION_PERIOD_START_EPOCH:
+            newly_opened_requests += 1
+        if request.request_status == RequestStatus.OPEN:
+            open_requests += 1
+        if request.request_status in [
```

Comment:
The list of request status states could be defined above and used here and for the newly created requests

Reply:
done